setup.py

```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```

Comment:
Sure, but this makes NO difference in terms of functionality. So why would we roll out a new RPM specifically for this?